src/screens/ConversationScreen.js

```diff
@@ -27,18 +28,98 @@ export default function ConversationScreen({ route, navigation }) {
       return <Text>No Chatbot Found with name '{chatbotName}'</Text>;
     }
   };
-
+  const [conversations, setConversations] = useState([]);
+  const [messages, setMessages] = useState([])
+  useEffect(() => {
+    fetchConversations();
+    if (user !==  null) {
+      setLoading(false);
+      console.log("USER", user)
+  }
+  }, [user]);
+  async function fetchConversations() {
+    try {
+        const { data, error } = await supabase
+        .from('conversations')
+        .select('*');
+        if (error) {
+        console.error('Error fetching conversations:', error.message);
+        return;
+        }
+        if (conversations) {
+          setConversations(data)
+          console.log("DATA", JSON.stringify(data, null, 4))
+          setMessages(data[0].messages);
+        }
+    } catch (error) {
+        console.error('Error fetching conversations:', error.message);
+    }
+  }
+  const handleInserts = (payload) => {
+    console.log('Change received!', JSON.stringify(payload, null, 4))
+    // if (payload.new.messages[0]._id === messages[0]._id) {
+    //   console.log()
+    // }
+    // console.log("PAYLOAD", payload.new.messages[0]._id)
+    // console.log("MESSAGES", messages[0]._id)
+    addNewMessage(payload.new.messages[0])
+  }
+  // Listen to inserts
+  supabase
+    .channel('conversations')
+    .on('postgres_changes', { event: 'UPDATE', schema: 'public', table: 'conversations' }, handleInserts)
+    .subscribe()
+  
+const addNewMessage = (newMessages) => {
+  setMessages((previousMessages) => {
+    // console.log("PREVIOUS MESSAGES:", previousMessages);
+    // console.log("NEW MESSAGE:", newMessages);
+    return GiftedChat.append(previousMessages, newMessages);
+  });
+};
+const onSend = useCallback((messages = []) => {
+  addNewMessage(messages);
+}, []);
+async function postConversations(newMessages) {
+  const allMessages = [newMessages[0],...messages];
+  const { data, error } = await supabase
+  .from('conversations')
+  .update({ messages: allMessages })
+  .eq('id', "areli_allison")
```

Comment:
This is a hardcoded value, please make sure to add a description to your PR explaining why this is here / what it should be. This way someone looking to build on / polish up your code knows what's going on.